package.json

```diff
@@ -9,11 +9,12 @@
   },
   "dependencies": {
     "babel-eslint": "4.1.8",
-    "eslint": "codeclimate/eslint.git#d24d0b",
-    "eslint-config-airbnb": "^5.0.0",
+    "eslint": "^2.2.0",
```

Comment:
Watch out for the ESLint 2.3.0 issue https://github.com/eslint/eslint/issues/5476